drivers/gles3/storage/config.cpp

```diff
@@ -81,6 +81,7 @@ Config::Config() {
 	astc_supported = extensions.has("GL_KHR_texture_compression_astc") || extensions.has("GL_OES_texture_compression_astc") || extensions.has("GL_KHR_texture_compression_astc_ldr") || extensions.has("GL_KHR_texture_compression_astc_hdr");
 	astc_hdr_supported = extensions.has("GL_KHR_texture_compression_astc_hdr");
 	astc_layered_supported = extensions.has("GL_KHR_texture_compression_astc_sliced_3d");
+	float_texture_linear_supported = extensions.has("GL_OES_texture_float_linear");
```

Comment:
This check should be moved below into the `if(RasterizerGLES3::is_gles_over_gl()) branch`. Because it is only implemented for GLES platforms (mobile + webgl). On desktop it will always be false despite the fact that desktop always supports RGB32F

It should always be true when `RasterizerGLES3::is_gles_over_gl()` is true and should be checked when it is false. 

Basically the same logic as `float_texture_supported`.